docs/led-demo-sequence.md

```diff
@@ -82,15 +82,15 @@ var demoSequence = [{
 function execute(step) {
 
   // Grab everything we need for this step
-  var method = demoSequence[step].method;
-  var args = demoSequence[step].args;
-  var duration = demoSequence[step].duration || 3000;
+  const method = demoSequence[step].method;
+  const args = demoSequence[step].args;
+  const duration = demoSequence[step].duration || 3000;
 
   // Just print out what we're executing
   console.log("led." + method + "(" + (args ? args.join() : "") + ")");
```

Comment:
Eventually I want to clean up this stuff as well, but you've already done a ton of work on these examples and I don't want to keep pushing back (so you can move on to another section)